### PR TITLE
Feature/process command

### PR DESCRIPTION
**Pull request by maxgrossman**
- processing scripts and pipeline to translate admin shapefile into PostGIS table

...the cleaning steps within the pipeline that handle geometry issues still need to be added. 

However, given the modularity of the pipeline and usefulness of feedback, its a good time for a pr. 
I will make the branch for cleaning a branch off of this branch.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+**/*/.DS_store
```
Comment (mileswwatkins) [inline .gitignore]: Small thing: remove `.DS_Store`s that have already been committed when this line came in.

File: processing/e-insert-tables.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  const properties = feature.properties;
+  const geometry = feature.geometry;
+  const statement = db.insert({
+    type: admin,
+    id: properties.id,
+    parent_id: properties.p_id,
+    geo: st.geomFromGeoJSON(geometry),
+    name_en: properties.en_name,
+    name_vn: ''
+  }).into(`${admin}-table`).toString();
```
Comment (mileswwatkins) [inline processing/e-insert-tables.js]: Won't all levels of boundaries live in the same admin table? That's how I have it set up, and how we did it for ORMA Philippines. With one table, it's easier to query, and self-joins make it as versatile as multiple tables (eg, `SELECT child.name_en FROM admin_areas AS parent JOIN admin_areas AS child ON parent.id=child.parent_id WHERE parent.id='12345'`). Plus, then it's portable to different countries (which would have different names for their admin levels).

**Review by mileswwatkins — COMMENTED** [review]
Looks good in general, especially if you've gone over it thoroughly with @kamicut. Left a couple notes.

File: processing/e-insert-tables.js (hunk 2 above)
Comment (maxgrossman) [reply processing/e-insert-tables.js]: For sure they can all live in the same table / they need to if that is how you have things set up! I'll make the changes to that script we're doing the right insert.

**Comment by maxgrossman** [comment]
remaining work for the pipe includes #6 and #7

**Comment by mileswwatkins** [comment]
There's not time right now to review this huge PR, nor can Max make changes to it. Devseed can make tweaks down the road if needed, and @sarahantos is also promising us a better source shapefile which might obviate this!

**Pull request merged**